client/screens/NavigationHeader.js

```diff
@@ -49,6 +49,12 @@ const MobileDropdown = ({ setIsMenuOpen }) => {
           <Text>Profile</Text>
         </View>
       </Link>
+      <Link href="setting">
```

Comment:
Does this appear on your end? Pretty sure we deprecated the NavigationHeader in favor of just Navigation.